src/primitives/transaction.h

```diff
@@ -216,8 +216,33 @@ class CTxOut
 
 struct CMutableTransaction;
 
+/**
+ * Transaction serialization format:
+ * - int32_t nVersion
+ * - std::vector<CTxIn> vin
+ * - std::vector<CTxOut> vout
+ * - uint32_t nLockTime
+ * - Optional<SaplingTxData> sapData
+ * - Optional<uint8_t> extraPayload
```

Comment:
typo: `extraPayload` is an optional **vector** of chars, not an optional char.

Reply:
typo fixed.